lib/line/bot/client.rb

```diff
@@ -363,6 +363,46 @@ def unlink_user_rich_menu(user_id)
         delete(endpoint_path)
       end
 
+      # To link a rich menus to multiple users at a atime
```

Comment:
=> `To link rich menus to multiple users at a time` ?

Reply:
Yes 🥇